### PR TITLE
added mtex2MML

### PR DESCRIPTION
**Pull request by dsisnero**
This is my first time creating a MINGW package.  Everything builds and works.

### CODE REVIEW

File: mingw-w64-lasem/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,55 @@
+_realname=lasem
```
Comment (Alexpux) [inline mingw-w64-lasem/PKGBUILD]: Add header for PKGBUILD. See any other PKGBUILD

File: mingw-w64-lasem/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+_REALNAME=LASEM
```
Comment (Alexpux) [inline mingw-w64-lasem/PKGBUILD]: Remove this

File: mingw-w64-lasem/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=0.4.3
+PKGVER=0_4_3
```
Comment (Alexpux) [inline mingw-w64-lasem/PKGBUILD]: Remove this

File: mingw-w64-lasem/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+	     "${MINGW_PACKAGE_PREFIX}-gtk-doc"
+	     "${MINGW_PACKAGE_PREFIX}-cairo"
+	     "${MINGW_PACKAGE_PREFIX}-pango"
+	     "${MINGW_PACKAGE_PREFIX}-libxml2"
+	     "flex"
+	     "bison"
+	     "intltool"
+	    )
+
+source=("https://github.com/GNOME/lasem/archive/${_REALNAME}_${PKGVER}.tar.gz")
```
Comment (Alexpux) [inline mingw-w64-lasem/PKGBUILD]: Replace ````${_REALNAME}_${PKGVER}```` with ````LASEM_${pkgver//./_}

File: mingw-w64-mtex2MML/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,52 @@
+_realname=mtex2MML
```
Comment (Alexpux) [inline mingw-w64-mtex2MML/PKGBUILD]: Add header

File: mingw-w64-mtex2MML/PKGBUILD [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+url="https://github.com/gjtorikian/mtex2MML/"
+license=('LGPL')
+license=('GPL' 'MPL' 'LGPL')
+makedepends=("${MINGW_PACKAGE_PREFIX}-python3"
+	     'bison'
+	     'flex'
+	     "${MINGW_PACKAGE_PREFIX}-cmake"
+	     "${MINGW_PACKAGE_PREFIX}-pkg-config"
+	    )
+source=("https://github.com/gjtorikian/mtex2MML/archive/v${pkgver}.tar.gz")
```
Comment (Alexpux) [inline mingw-w64-mtex2MML/PKGBUILD]: Download into named tarball

**Comment by dsisnero** [comment]
I am not sure why I am getting a missing dependency sometimes - 
 Missing dependencies:
  -> mingw-w64-x86_64-gtk-doc

from PKGBUILD
 "${MINGW_PACKAGE_PREFIX}-gtk-doc"

**Comment by dsisnero** [comment]
Ok, I made the changes.  One thing that I have a question on is the pkg-config file.  The package config file for lasem is named lasem-0.4.pc  .  Do I want to make a patch to change the name to lasem.pc ?

File: mingw-w64-mtex2MML/PKGBUILD [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+url="https://github.com/gjtorikian/mtex2MML/"
+license=('LGPL')
+license=('GPL' 'MPL' 'LGPL')
+makedepends=("${MINGW_PACKAGE_PREFIX}-python3"
+	     'bison'
+	     'flex'
+	     "${MINGW_PACKAGE_PREFIX}-cmake"
+	     "${MINGW_PACKAGE_PREFIX}-pkg-config"
+	    )
+source=("https://github.com/gjtorikian/${_realname}/archive/v${pkgver}.tar.gz")
```
Comment (Alexpux) [inline mingw-w64-mtex2MML/PKGBUILD]: Add ````${_realname}-${pkgver}.tar.gz::```` before URL

File: mingw-w64-lasem/PKGBUILD [hunk 8]
```diff
@@ -0,0 +1,54 @@
+# Maintainer: Dominic Sisneros <dsisnero@gmail.com>
+
+_realname=lasem
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=0.4.3
+pkgrel=1
+pkgdesc="Lasem aims to be a C/Gobject based SVG/Mathml renderer"
+arch=('any')
+url="Lasem aims to be a C/Gobject based SVG/Mathml renderer"
```
Comment (Alexpux) [inline mingw-w64-lasem/PKGBUILD]: Fix URL.

**Comment by Alexpux** [comment]
@dsisnero you not yet fix all comments

**Comment by dsisnero** [comment]
Ok, I think I finally changed everything requested.

**Pull request merged**